src/C4SoundSystem.cpp

```diff
@@ -256,20 +258,24 @@ bool C4SoundSystem::Instance::IsNear(const C4Object &obj2) const
 	return false;
 }
 
+// Finds an existing sound instance
+// Returns an array of matching files
 auto C4SoundSystem::FindInst(const char *wildcard, const C4Object *const obj) ->
 	std::optional<decltype(Sample::instances)::iterator>
 {
-	const auto wildcardStr = PrepareFilename(wildcard);
-	wildcard = wildcardStr.c_str();
+	for (const auto fallbackFileExtension : fileExtensions) {
```

Comment:
Code style: Move `{` to the beginning of the next line, as seen in other blocks.
Also, use a const reference instead of a const copy (`const auto &fallbackFileExtension`).